package.json

```diff
@@ -162,6 +162,7 @@
         "find-remove": "1.2.1",
         "jquery": "^3.4.1",
         "lockfile": "1.0.4",
+        "monaco-editor": "^0.52.2",
```

Comment:
We're already using `@monaco-editor/react` for schema compare; is this separate package.json entry necessary?